src/index.html

```diff
@@ -4,9 +4,469 @@
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="MOYO companys logo image"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#apple"
+              class="header__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#samsung" class="header__link">
+              samsung
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#smartphones" class="header__link">
+              smartphones
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#laptops"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#gadgets" class="header__link">
+              gadgets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#tablets" class="header__link">
+              tablets
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#photo" class="header__link">
+              photo
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#video" class="header__link">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <article class="catalog">
+          <article class="card catalog__card" data-qa="card">
+            <img
+              src="./images/imac.jpeg"
+              alt="apple a1419 imac 27 inch"
+              class="card__image"
+            >
+
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <span class="card__id">
+              Product code: 195434
+            </span>
+
+            <div class="card__reviews">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <span class="card__description">
```

Comment:
bad class name, card can have many descriptions, so its better to name it something like "card__reviews-tittle"